src/AdaProx.jl

```diff
@@ -272,6 +272,43 @@ function stepsize(rule::OurRule, (gamma1, gamma0), x1, grad_x1, x0, grad_x0)
     return (gamma, sigma), (gamma, gamma1)
 end
 
+
+
+struct OurRulePlus{R}
+    gamma::R
+    xi::R
+    nu::R
+    r::R
+end
+
+function OurRulePlus(; gamma = 0, nu = 1, xi = 1, r = 1/2)
```

Comment:
Why use the default `gamma = 0`? I guess it’s required positive to initialize the stepsize sequence, so you can also not give it a default

Reply:
Do we keep it in the same style as OurRule (to be updated with added sigma for the primal-dual version), or shall I remove gamma already?